### ISSUE TITLE
Creating PHP WebApp Fails

### ISSUE THREAD

**Issue by hawkeyes0v0** [Azure/azure-cli#12114]
## Describe the bug

**Command Name**
`az webapp create`

**Errors:**
```
update_site_config() takes 2 positional arguments but 3 were given
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 657, in _run_job
    cmd_copy.exception_handler(ex)
...
cli/core/commands/__init__.py, ln 306, in __call__
    return self.handler(*args, **kwargs)
azure/cli/core/__init__.py, ln 493, in default_command_handler
    return op(**command_args)
cli/command_modules/appservice/custom.py, ln 135, in create_webapp
    match['setter'](cmd, match, site_config)
TypeError: update_site_config() takes 2 positional arguments but 3 were given
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp create --resource-group {} --name {} --plan {} --runtime {} --deployment-local-git`

## Environment Summary
```
Linux-4.15.0-1067-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.81

```
## Additional Context

Possibly related to:
https://github.com/Azure/azure-cli/issues/12109

**Comment by arrownj** [comment]
@hawkeyes0v0 , thanks for reporting, we are looking at it.

**Comment by panchagnula** [comment]
Same as #12159, please use the EdgeBuilds the fix will be in 2.0.82
